src/components/login/OIDCs.tsx

```diff
@@ -1,27 +1,61 @@
 import { Box } from '@mui/material';
-import GoogleAuthButton from 'components/login/OIDCs/Google';
+import { Provider } from '@supabase/supabase-js';
+import { useClient } from 'hooks/supabase-swr';
+import { useSnackbar } from 'notistack';
+import GoogleButton from 'react-google-button';
+import { useIntl } from 'react-intl';
 import useConstant from 'use-constant';
 
-interface Props {
-    onError: (messageID: string) => void;
-}
-
-function OIDCs({ onError }: Props) {
+function OIDCs() {
     const redirectTo = useConstant(
         () => `${window.location.origin}` // `${window.location.origin}${routeDetails.registration.path}`
     );
 
+    const supabaseClient = useClient();
+    const intl = useIntl();
+
+    const { enqueueSnackbar } = useSnackbar();
+
+    const loginFailed = (key: Provider) => {
```

Comment:
All OIDCs can use the same general pattern when messaging that they were unable to login.